Update GoogleTests.java

AleksandrZhuravel opened the pull request:
1. Selectors "q" and "html" replaced by class selectors.

AleksandrZhuravel commented:
1. Selectors "q" and "html" replaced by class selectors.

kristina71 commented on src/test/java/tests/GoogleTests.java:
```diff
@@ -14,9 +14,9 @@ void selenideSearchTest() {
 
         // Ввести Selenide в поиск
-        $(byName("q")).setValue("Selenide").pressEnter();
+        $(".gLFyf").setValue("Selenide").pressEnter();
```
`$(".gLFyf")` Perhaps, This is a dynamic class that can be generated when rendering a page. Also, this locator is intuitively incomprehensible when reading code. It is better not to get attached to it.

kristina71 commented on src/test/java/tests/GoogleTests.java:
```diff
@@ -14,9 +14,9 @@ void selenideSearchTest() {
 
         // Проверить, что Selenide появился в результатах поиска
-        $("html").shouldHave(text("ru.selenide.org"));
+        $(".iUh30").shouldHave(text("ru.selenide.org"));
```
the same problem

kristina71 reviewed (changes requested):
And also I don't see the Yandex test. You need to create one more test (https://qa.guru/pl/teach/control/lesson/view?id=161420799&editMode=0 - step 3)

kristina71 commented on src/test/java/tests/GoogleTests.java:
```diff
@@ -14,9 +14,22 @@ void selenideSearchTest() {
+     @Test
+    void selenideSearchYandexTest() {
+        // Открыть google
+        open("https://google.com");
```
Why do you open google.com? You should open yandex.ru and create any test

For example:
`@Test
    void emptySearch(){
        open("https://yandex.ru");
        $(".input__control").setValue("").pressEnter();
        $(".misspell__message").shouldHave(text("Задан пустой поисковый запрос"));
    }`

kristina71 commented on src/test/java/tests/GoogleTests.java:
```diff
@@ -14,9 +14,22 @@ void selenideSearchTest() {
+        open("https://google.com");
+
+        // Ввести Selenide в поиск
+        $(byName("q")).setValue("Selenide").pressEnter();
+
+        // Проверить, что Selenide появился в результатах поиска
+        $("html").shouldHave(text("ru.selenide.org"));
```
What is the difference between this test and the previous one?

AleksandrZhuravel commented:
You were right about my request. I reversed my false changes and added new Yandex-test. 
I also reviewed the code in accordance with the wishes of Stanislav Vasenkov and added the UTF-8 encoding to the file "build.gradle".

kristina71 commented on src/test/java/tests/GoogleTests.java:
```diff
@@ -10,26 +10,24 @@
 
-        // Проверить, что Selenide появился в результатах поиска
-        $("html").shouldHave(text("ru.selenide.org"));
+        $("html").shouldHave(text("ru.selenide.org"));// Проверить, что Selenide появился в результатах поиска
```
better use specific locator like ".organic__subtitle b"

AleksandrZhuravel commented:
I changed the locator to a locator by class (different from yours) and renamed Java class. You should rename GoogleTests.java according to the Java class name.
